simphony/cuds/lattice.py

```diff
@@ -149,17 +151,18 @@ def data(self, value):
 
 
 def make_hexagonal_lattice(name, h, size, origin=(0, 0)):
-    """Create and return a 2D hexagonal lattice.
+    """Create and return a 2D hexagonal lattice embedded on the XY-plane
+    in 3D.
 
     Parameters
     ----------
     name : str
     h : float
-        lattice spacing.
-    size : 2 x int
-        number of lattice nodes (in each axis direction).
-    origin : 2 x float (default value = (0,0))
-        lattice origin.
+        lattice spacing
+    size : int[2]
+        Number of lattice nodes in each axis direction.
+    origin : float[2], default value = (0, 0)
```

Comment:
You're right. Docstrings already say that the lattice will be embedded in 3D so the origin must be in 3D.